src/components/header/Header.tsx

```diff
@@ -1,2 +1,57 @@
+import cartIcon from '../../assets/cart.png';
```

Comment:
suggest you create import alises for component, assets, etc
https://dev.to/tilly/aliasing-in-vite-w-typescript-1lfo